app/controllers/admin/requests_controller.rb

```diff
@@ -76,4 +82,24 @@ def paginate_requests requests
       pagy(requests, items: Settings.number_20)
     end
   end
+
+  def check_out_of_stock selected_books
+    out_of_stock_books = Book.available?(selected_books)
+    if out_of_stock_books.present?
+      flash[:warning] =
+        "#{out_of_stock_books.join(', ')} #{t('noti.books_out_of_stock')}"
+      @is_error = true
+      true
+    else
+      false
+    end
```

Comment:
```suggestion
    return false if out_of_stock_books.blank?
      flash[:warning] =
        "#{out_of_stock_books.join(', ')} #{t('noti.books_out_of_stock')}"
      @is_error = true
      true

```
tên hàm thêm dấu ? nhé nếu kết quả trả về true hoặc false